app/renderer/command.js

```diff
@@ -2358,10 +2361,24 @@ const execute = (com, settingsFile = null) => {
     // Remove all redundant spaces
     // Allow commands prefixed with :
     // And return if the command is empty
-    const commandStr = com.replace(/^[\s|:]*/, "").trim().replace(/ +/g, " ")
+    let commandStr = com.replace(/^[\s|:]*/, "").trim().replace(/ +/g, " ")
     if (!commandStr) {
         return
     }
+
+    // Don't "use current" on holdUseCommands, commands like 'command' or 'map'
```

Comment:
This section is good. Can I suggest that we add more variants of the url? Most importantly domain (origin without protocol, I think it's named `.host` on the url spec). Origin can stay, as that is also useful if you do want the protocol. And just move the replacement functions to this file, they are not actions and shouldn't be treated as such, maybe we have add a paragraph to the command help to explain these templated `use` strings?